cloudwatch_dashboard_rds/main.tf

```diff
@@ -47,7 +47,7 @@ EOM
 
 resource "aws_cloudwatch_dashboard" "main" {
   count = var.enabled
-  
+
```

Comment:
nitpick in that we should be consistent how we use new lines inside resource blocks and the OCD in me wishes that we'd even alphabetize things or have them organized in some fashion.

Reply:
I agree, it looks like this was changed because it was a newline and a space, so Terraform removed the dangling space.  It also looks like it will always add 1 newline after `count` to show how many of X resource there is